### ISSUE TITLE
Shiga-toxin-related hemolytic-uremic syndrome 

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6870]
**Preferred term label:**
Shiga-toxin-related hemolytic-uremic syndrome 

**Synonyms**
typical HUS, STEC-HUS, Shiga-toxin-associated HUS

**Definition (free text, please give PubMed ID)**
A form of hemolytic-uremic syndrome induced by Shiga-toxin, typically a few days after gastrointestinal infection with Shiga-toxin producing Escherichia coli. Shiga toxin binds cell membrane glycolipid Gb3 on the endothelium and causes damage to or activation of endothelium, red cells, and platelets and thereby initiates the disease process.
 PMID: 19846853, PMID: 28416508
**Parent term (use hpo.jax.org/app)**
Hemolytic-uremic syndrome HP:0005575

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
I would consider this a disease rather than a phenotypic feature.
The corresponding MONDO term is 
https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0019536
